core/src/com/fourinachamber/fortyfive/game/card/Card.kt

```diff
@@ -777,15 +785,15 @@ class CardActor(
         onClick {
             if (!inSelectionMode) return@onClick
             // UGGGGGLLLLLLYYYYY
-            FortyFive.currentGame!!.selectCard(card)
+//            FortyFive.currentGame!!.selectCard(card)
         }
         onHoverEnter {
             if (!playSoundsOnHover) return@onHoverEnter
             SoundPlayer.situation("card_hover", screen)
         }
         onTouchEvent { event, _, _ ->
```

Comment:
please don't use onTouch, onHoverEnter and onClick, use onFocus and onSelect instead